sdk/communication/communication-rooms/test/public/roomsClient.spec.ts

```diff
@@ -128,6 +128,26 @@ describe("RoomsClient", function () {
       await client.updateRoom(roomId, options);
     });
 
+    it("successfully list rooms", async function () {
```

Comment:
`xit` will disable test so you don't have to comment everything out; or add a `this.skip()`

Reply:
nice will do